files/settings-default.cfg

```diff
@@ -267,6 +267,11 @@ use magic item animations = false
 # Don't use race weight in NPC movement speed calculations
 normalise race speed = false
 
+# Percent of soul gem charge, which can be used to enchant multiple projectiles:
+# count = gemCharge / enchantmentPower * projectilesEnchantMultiplier
```

Comment:
Adjusts the number of projectiles you can enchant at once.
Same suggestions as above.